wdl-ast/src/v1/expr.rs

```diff
@@ -2125,6 +2169,116 @@ impl LiteralString {
 
         None
     }
+
+    /// Strips leading whitespace from a multi-line string.
+    ///
+    /// This function will remove leading whitespace from each line of a
+    /// multi-line string and parse line continuations. Single or double
+    /// quoted strings will return `None`.
+    pub fn strip_whitespace(&self) -> Option<Vec<StrippedStringPart>> {
+        if self.kind() != LiteralStringKind::Multiline {
+            return None;
+        }
+
+        let mut result = Vec::new();
+
+        // Parse the string parts and remove line continuations.
+        // We also remove the first and last lines of the string.
+        // Placeholders are copied as-is.
+        for (i, part) in self.parts().enumerate() {
+            match part {
+                StringPart::Text(text) => {
+                    let parsed = remove_line_continuations(text.as_str());
+                    let mut reconstructed = Vec::new();
+                    for (j, line) in parsed.lines().enumerate() {
+                        if i == 0 && j == 0 {
+                            let trimmed = line.trim_start();
+                            if !trimmed.is_empty() {
+                                reconstructed.push(trimmed.to_string());
+                            }
+                            continue;
+                        }
+                        reconstructed.push(line.to_string());
+                    }
+
+                    result.push(StrippedStringPart::Text(reconstructed.join("\n")));
+                }
+                StringPart::Placeholder(placeholder) => {
+                    result.push(StrippedStringPart::Placeholder(placeholder));
+                }
+            }
+        }
+
+        if let Some(StrippedStringPart::Text(text)) = result.last_mut() {
+            if let Some(index) = text.rfind(|c| !matches!(c, ' ' | '\t')) {
+                text.truncate(index + 1);
+            } else if text.chars().all(|c| matches!(c, ' ' | '\t')) {
```

Comment:
I think this `else if` could be an `else`, right?

Reply:
Ah yes. You're right!